src/main/java/nl/workingtalent/bookrental/controller/UserController.java

```diff
@@ -26,24 +27,26 @@ public class UserController {
 	@Autowired
 	private IUserRepository repo;
 	
+	@Autowired
+	private EmailService emailService;
+	
 	@PostMapping("user/create")
 	public String createUser(@RequestHeader(name = "Authorization") String token, @RequestBody NewUserDto newUserDto) {
+		
 		User loggedInUser = repo.findByToken(token);
 
+		// Check if user had Admin rights
 		if (loggedInUser == null || !loggedInUser.isAdmin()) {
 			return "No permission";
 		}
 		
 		// Check if user already exists in database by email
```

Comment:
Kan korter -> if (repo.findByEmail(newUserDto.getEmail())) return "Email already exists";

Je hoeft het niet op te slaan als object in dit geval, en in theorie zou het ook moeten werken zonder != null, maar is opzich best nice voor readability